packages/fela-monolithic/README.md

```diff
@@ -5,7 +5,7 @@
 
 The monolithic enhancer will use unique class names instead of atomic ones.
 These generated class names are not re-usable like the atomic design but allows you to debug and modify styles with ease.
-Every ruleset will have it's own unique class - this means that a new class will be generated if you are using props and they change.
+Every ruleset will have it's own unique class - this means that a new class will be generated if you are using props and they change. If you want to fix the class name, you can add property `className` into your rule set. This can be useful if you want to generate an external stylesheet that's human readable, re-usable and can be used on non-JS projects.
```

Comment:
Would be cool to add an example to the Usage section here? So ppl understand why and what happens :P